src/judging_round_display.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+
+
+class JudgingRoundDisplay extends React.Component {
+    state = {
+        data: {},
+    }
+
+    componentDidUpdate(prevProps) {
+	const judging_round_id = this.props.judging_round
+	if (judging_round_id &&
+	      prevProps.judging_round !== this.props.judging_round) {
+	    this.fetch_data()
+	}
+    }
+
+    fetch_data() {
+	fetch("http://localhost:8000/api/v1/judging_round/" +
+	      judging_round_id + "/",
+	      {credentials: "include", mode: "cors"})
+	    .then(res => res.json()).then(data => {
+		this.setState({ data: data })
```

Comment:
When the keys and values are the same literal string, you can shorten this to:
```
this.setState({ data })
```

Reply:
Done.